spec/unit/pluginHandler/windows.spec.js

```diff
@@ -304,6 +309,47 @@ describe('windows project handler', function () {
                 xpath = 'Reference[@Include="dummy6"]/HintPath';
                 validateInstalledProjects('framework', frameworks[5], xpath, ['windows', 'windows10', 'phone']);
             });
+
+            it('with .winmd and .dll files', function() {
```

Comment:
Does this spec check that dll files have been actually copied? ([according to this](https://github.com/apache/cordova-windows/pull/219/files#diff-7bcd02ec7f03ffbb25512bd1f4a0d388R99))